packages/xrpc-server/src/server.ts

```diff
@@ -109,6 +115,12 @@ export class Server {
       } catch (e) {
         throw new InvalidRequestError(String(e))
       }
+
+      // @TODO move to middleware
```

Comment:
I believe it's the `raw()` middleware that does this 👍 Should jibe nicely with the reorganization in #384.

Reply:
Maybe you already gave this a whirl, but I think you will need to set its [`type` option](https://expressjs.com/en/api.html#express.raw) for our use-case.